pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/ComputeStep.java

```diff
@@ -249,13 +321,20 @@ private Schema getAvroSchema(ComputeFieldType type) {
       case STRING:
         schemaType = Schema.Type.STRING;
         break;
+      case INT8:
```

Comment:
This could be a FIXED type in Avro. Using INT for simplicity but let me know if it is better to change it. 

If I don't know about the pulsar types, then by default expectation for users to use Avro's INT or LONG.

Reply:
I think it's OK to map to INT atm.
At least it's not semantically wrong. We can change later.

Reply:
Also AVRO int and long are encoded using variable zig-zag encoding so it's not even sure that you gain bytes by using a fixed type. It depends on the application.